<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMain</class>
 <widget class="QWidget" name="frmMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1508</width>
    <height>797</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Liste des allocataires</string>
  </property>
  <widget class="QLineEdit" name="leTxtSaisie">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="lstBDD">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>281</width>
     <height>681</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tvDatas">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>1161</width>
     <height>721</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">alternate-background-color: rgb(221, 221, 110);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
